control/control.go

```diff
@@ -218,7 +218,22 @@ func New(cfg *Config) *pluginControl {
 	}).Debug("metric catalog created")
 
 	// Plugin Manager
-	c.pluginManager = newPluginManager(OptSetPprof(cfg.Pprof), OptSetTempDirPath(cfg.TempDirPath))
+	managerOpts := []pluginManagerOpt{
+		OptSetPprof(cfg.Pprof),
+		OptSetTempDirPath(cfg.TempDirPath),
+	}
+	if cfg.IsTLSEnabled() {
+		if cfg.RootCertPaths != "" {
```

Comment:
Applicable to any part in the code referring to "root cert" - consider changing to "ca cert", "certificates authority cert", or "root ca cert". Just using "root cert" might be misleading.